data/registry/instrumentation-dotnet-nservicebus.yml

```
@@ -0,0 +1,22 @@
+title: NServiceBus OpenTelemetry Integration # the name of your project
+registryType: instrumentation # the type of integration; is this an exporter, plugin, API package, or something else?
```

Please remove the comments from here and elsewhere in the file:

```suggestion
title: NServiceBus OpenTelemetry Integration
registryType: instrumentation
```